include/treasures.h

```diff
@@ -0,0 +1,35 @@
+#ifndef GUARD_TREASURES_H
+#define GUARD_TREASURES_H
+
+#include "global.h"
+
+struct Treasures {
+    u8 chestFields[0x10];
+    u32 shardField;
+    u32 unk10_2;
+    u32 unk10_3;
```

Comment:
```suggestion
    u32 unk14;
    u32 unk18;
```
`unk_a_b` is used for bitfields